code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/github/GithubUtils.java

```diff
@@ -125,4 +148,58 @@ public String getTeamOrgById(String id) {
         Map<String, String> teamToOrg = (Map<String, String>) jsonData.get("teamToOrg");
         return  teamToOrg.get(id);
     }
+
+    public boolean isAllowed(List<String> idList, Set<ExternalId> externalIds){
+        boolean whiteListed = getWhitelistedUser(idList) != null;
```

Comment:
This method was written just to gram the first id from the list that is whitelisted. I didn't change it from the original impl as I didn't want to spend time reading it and seeing it if needed to be changed as it already worked. I can refactor it to make it better but didn't see the need.